new script PathToWindowOrDoor

groch opened the pull request:
this one helps to convert 2 point path to new door&windows

Alicekb commented:
Hey Groch, There are some merge conflicts that need to be resolved before I can go ahead and merge these changes.

groch commented:
Hi, i think my other commit introduced a folder creation that interract with it, i see threads talking that similar issues happen when renaming files and i think it's kind of that

i updated my fork, rebased my branch and merged from master, and it still says no change to commit ...
can you point me how i can help ? ^^'

groch reviewed (commented):
validating all the changes
